science/ngspice/Portfile

```diff
@@ -3,11 +3,11 @@
 PortSystem      1.0
 
 name            ngspice
-version         36
+version         43
 revision        0
 license         BSD
 categories      science cad
-maintainers     nomaintainer
+maintainers     {gmail.com:degnan.68k @Brian Degnan} openmaintainer
```

Comment:
That doesn't look like your github handle.

Reply:
I'll address that to make it @bpdegnan